libs/ml/include/ml/ops/abs.hpp

```diff
@@ -90,13 +90,14 @@ class Abs : public Ops<T>
     assert(error_signal.size() == inputs.at(0)->size());
 
     TensorType return_signal(inputs.at(0)->shape());
+    using DataType = typename TensorType::Type;
```

Comment:
move this to be with all the other using's as a public member of the class

Reply:
fixed.